core/src/components/title/title.ionic.scss

```diff
@@ -0,0 +1,39 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "title.common";
+
+// Ionic Title
+// --------------------------------------------------
+
+:host {
+  @include globals.typography(globals.$ionic-heading-h6-medium);
+  @include globals.padding(var(--padding-top), 0, var(--padding-bottom));
+
+  box-sizing: border-box;
+  pointer-events: none;
+}
+
+:host-context(ion-toolbar) {
+  @include globals.position(0, null, null, 0);
+  position: absolute;
+
+  width: 100%;
+  height: 100%;
+
+  transform: translateZ(0);
+
+  z-index: -1; // TODO(ROU-10853): replace this value with a layer token. This is here to prevent the title from blokcing interaction with buttons.
+}
+
+// Title: Small
+// --------------------------------------------------
+
+:host(.title-small) {
+  @include globals.typography(globals.$ionic-heading-h6-medium);
```

Comment:
Is this necessary? It is the same font size as the default size toolbar.

Reply:
It was just to separate the styles, as in the future the default might be different, but I can remove this for now 👍